main.cpp

```diff
@@ -60,18 +60,34 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
+    int count = 0;
+    int dots = 5;
+    int cities = 3;
+    char alpha = 'b';
+    char grade = 'C';
+    char rating = 'A';
+    bool lieDetectorTest = true;
+    bool containsLead = false;
+    bool underground = true;
+    float firstNum = 23.3f;
+    float anotherNum = 0.002f;
+    float difference = 1.2323f;
+    double smallNum = 3E-6;
+    double timeRemain = 255.23;
+    double halfTime = .5;
 
     
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(count, dots, cities, alpha, grade, rating, lieDetectorTest, containsLead, underground, firstNum, anotherNum, difference, smallNum, timeRemain, halfTime);

```

Comment:
call `ignoreUnused` once, not twice.
D.R.Y.  Don't Repeat Yourself
- [ ] revise